impeller/entity/contents/filters/gaussian_blur_filter_contents_unittests.cc

```diff
@@ -225,5 +227,82 @@ TEST_P(GaussianBlurFilterContentsTest, CalculateUVsSimple) {
   }
 }
 
+TEST_P(GaussianBlurFilterContentsTest, TextureContentsWithDestinationRect) {
+  TextureDescriptor desc = {
+      .storage_mode = StorageMode::kDevicePrivate,
+      .format = PixelFormat::kB8G8R8A8UNormInt,
+      .size = ISize(100, 100),
+  };
+
+  std::shared_ptr<Texture> texture = MakeTexture(desc);
+  auto texture_contents = std::make_shared<TextureContents>();
+  texture_contents->SetSourceRect(Rect::MakeSize(texture->GetSize()));
```

Comment:
@bdero is this something I have to worry about?  I don't think your tester has a way of modifying this.

Reply:
Having a smaller source rect will result in the snapshot performing a subpass instead of just passing through the texture, but the snapshot will look otherwise very similar as far as the blur is concerned.